Library/Homebrew/sandbox.rb

```diff
@@ -8,18 +8,6 @@ def self.available?
     OS.mac? && File.executable?(SANDBOX_EXEC)
   end
 
-  # there are times the sandbox cannot be used.
-  def self.auto_disable?
-    @auto_disable ||= ARGV.interactive? || ARGV.debug?
-  end
-
-  def self.print_autodisable_warning
-    unless @printed_autodisable_warning
-      opoo "The sandbox cannot be used in debug or interactive mode."
-      @printed_autodisable_warning = true
-    end
-  end
```

Comment:
I haven't looked into this, but could there be other reasons why the sandbox cannot/couldn't be used in debug/interactive mode? I.e. what is the original reason for this limitation?